safekeeper/src/lib.rs

```diff
@@ -61,6 +61,7 @@ pub struct SafeKeeperConf {
     pub remote_storage: Option<RemoteStorageConfig>,
     pub max_offloader_lag_bytes: u64,
     pub backup_runtime_threads: Option<usize>,
+    pub backup_parallel_jobs: usize,
```

Comment:
Pedantic mode on: strictly speaking these are concurrent jobs, not parallel

Reply:
True, but most of the time is spent waiting for disk data or waiting for socket data to be transmitted, and that is done in parallel by the kernel.